base/v0_6_exp/validate.go

```diff
@@ -29,10 +30,24 @@ func (rs Resource) Validate(c path.ContextPath) (r report.Report) {
 	if rs.Local != nil {
 		sources++
 		field = "local"
+		config := *rs.Local
+		_, report, err := exp.Parse([]byte(config))
+		if err != nil {
+			r.Merge(report)
```

Comment:
We cannot just merge this, the path fields in the report are ign specific. We need to take the report data and create a mapper function which translates the data into butane's paths

Reply:
no problem, I've made a start at the mapper function, I'm not too familiar with what exactly needs to be mapped but will look into it!